src/components/preferences/General.js

```diff
@@ -0,0 +1,274 @@
+                        }
+                        color="primary"
+                        inputProps={{
+                            "aria-label": formatMessage(
+                                intl,
+                                "analysisStatusEmailLbl"
+                            ),
+                        }}
+                    />
+                </GridLabelValue>
+                <GridLabelValue
+                    label={getMessage("urlImportEmailLbl")}
+                    labelVariant="body1"
+                >
+                    <Field
+                        id={build(
+                            baseId,
+                            ids.URL_IMPORT_EMAIL_NOTIFICATION_SWITCH
+                        )}
+                        component={FormSwitch}
+                        name={
+                            prefConstants.keys.ENABLE_IMPORT_EMAIL_NOTIFICATION
+                        }
+                        color="primary"
+                        inputProps={{
+                            "aria-label": formatMessage(
+                                intl,
+                                "urlImportEmailLbl"
+                            ),
+                        }}
+                    />
+                </GridLabelValue>
+            </Grid>
+            <Divider className={classes.dividers} />
+            <Typography variant="h6" className={classes.sectionHeader}>
+                {getMessage("resetHPCTokenLbl")}
+            </Typography>
+            <Grid container spacing={3} className={classes.grid}>
+                <GridLabelValue
+                    label={getMessage("resetHPCTokenPrompt")}
+                    labelVariant="body1"
+                >
+                    <Button
+                        color="primary"
+                        variant="outlined"
+                        disabled={requireAgaveAuth}
+                        onClick={() =>
+                            resetHPCToken({
+                                systemId: constants.AGAVE_SYSTEM_ID,
+                            })
+                        }
+                    >
+                        {getMessage("resetHPCTokenLbl")}
+                    </Button>
+                </GridLabelValue>
+            </Grid>
+            {defaultOutputFolder && (
+                <SelectionDrawer
+                    id={build(baseId, ids.FOLDER_SELECTION_DRAWER)}
+                    open={openFileBrowser}
+                    onClose={() => setOpenFileBrowser(false)}
+                    startingPath={defaultOutputFolder}
+                    acceptedType={ResourceTypes.FOLDER}
+                    onConfirm={(selection) => {
+                        setOpenFileBrowser(false);
+                        onNewDefaultOutputFolder(selection);
+                    }}
+                    baseId={baseId}
+                    multiSelect={false}
+                />
+            )}
+        </>
```

Comment:
Without the `defaultOutputFolder` condition, for some weird reason, the `SelectionDrawer` will always open the users home folder and ignore the `startingPath`. I haven't debugged it entirely yet.